src/data/duke.txt

```diff
@@ -0,0 +1,4 @@
+3
+[T][X] yyy
+[D][X] kdayyy  (by: 2021-12-21)
+[T][X] skdakda
```

Comment:
Perhaps omit this file from commit, or keep it empty so when others run duke they do not inherit your tasks?